src/apps/vpn/commands/commandui.cpp

```diff
@@ -680,7 +681,6 @@ int CommandUI::run(QStringList& tokens) {
 #endif
 
     KeyRegenerator keyRegenerator;
-
```

Comment:
I like this empty line :)

Reply:
LOL, I removed it by mistake, because initially I added the start of the timer here. Let me add it back!